style.css

```diff
@@ -370,3 +342,76 @@ div .taggs {
   border-radius: 6px;
   color: #396df2;
 }
+
+.frameworks {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 38px 0 10px;
+}
+
+.contact {
+  width: 98%;
+  background: #6070ff;
+  background-image: url(images/contact/layers.svg);
+  background-repeat: no-repeat;
+  background-position-x: 135px;
+  background-position-y: 10px;
+  border-top-left-radius: 6rem;
+}

```

Comment:
- [ ] Please kindly make sure your design stick as much as possible to the Figma design. Your background image is not aligned properly, kindly fix their position to the right 🙃 😉

Your design

<img width="299" alt="Screen Shot 2022-11-02 at 00 00 42" src="https://user-images.githubusercontent.com/74501523/199350682-216ddc8a-7ecf-4d2f-bfb6-79aba7aff296.png">

Figma design

<img width="303" alt="Screen Shot 2022-11-02 at 00 01 32" src="https://user-images.githubusercontent.com/74501523/199350729-58ea9a82-6344-4c37-97ec-5834b52b02d5.png">